compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -152,15 +166,32 @@ private String getName(TypeMirror type) {
             .build())
         .addModifiers(PUBLIC, FINAL)
         .addSuperinterface(ClassName.get("bullet", "ObjectGraph"))
-
         .addField(elementName, "component", PRIVATE, FINAL)
-
         .addMethod(MethodSpec.constructorBuilder()
             .addModifiers(PUBLIC)
             .addParameter(elementName, "component", FINAL)
             .addCode("this.component = component;\n")
             .build());
 
+    if (membersInjectionMethodsMap.size() > 0) {
+      classBuilder.addField(ClassName.get("bullet.impl", "ClassIndexHashTable"), "classIndexHashTable", PRIVATE, STATIC, FINAL);
+    }
+
+    // Generate the ClassIndexHashTable if there are classes to inject.
+    if (membersInjectionMethodsMap.size() > 0) {
+      // ClassIndexHashTable size should be at least 30% larger and a prime number.
+      int classIndexHashTableSize = getNextPrime((int) Math.ceil(membersInjectionMethodsMap.size() * 1.3));
```

Comment:
I appreciate the clarification. Fix commit: https://github.com/WeMakeBetterApps/bullet/commit/05b94baf97df6fea4b5665bba3fb800d0b2328a1